lib/elixir/lib/module.ex

```diff
@@ -1201,6 +1201,6 @@ defmodule Module do
   defp assert_not_compiled!(fun, module) do
     open?(module) ||
       raise ArgumentError,
-        "could not call #{fun} on module #{inspect module} because it was already compiled"
+        "could not call #{fun} in module #{inspect module} because it was already compiled"
```

Comment:
Hey, thanks for the feedback. Your understanding of the system is no doubt much greater than mine. If I understand you correctly, should this actually say:
`"could not call #{fun} in with argument #{inspect module} because it is already compiled"`?